lib/improver/tests/ensemble_copula_coupling/ensemble_copula_coupling/test_EnsembleReordering.py

```diff
@@ -595,6 +604,17 @@ def test_basic(self):
         self.assertArrayAlmostEqual(
             result.coord("realization").points, [0, 1, 2])
 
+    def test_works_for_different_percentile_coord(self):
```

Comment:
We need to test the reordering plugin with a cubelist as well, as I think this would have captured the mistake I mentioned in the cube/cubelist differentiation above.

Reply:
Added